src/sentry/utils/snuba.py

```diff
@@ -1221,6 +1228,29 @@ def _snuba_query(
                 raise SnubaError(err)
 
 
+def _raw_delete_query(
+    request: Request, headers: Mapping[str, str]
+) -> urllib3.response.HTTPResponse:
+    query = request.query
+    if not isinstance(query, DeleteQuery):
+        raise ValueError(
+            f"Expected request to contain a DeleteQuery but it was of type {type(request.query)}"
+        )
+
+    # Enter hub such that http spans are properly nested
+    with timer("delete_query"):
+        referrer = headers.get("referer", "unknown")
```

Comment:
Do headers have the key "referer" or "referrer" with an extra r? If it's the former, then would it be a big change to make referrer consistent everywhere?

Reply:
i looked into this a bit and its bc of someones design of a headers object. it may be possible to change but it wouldnt be trivial and would potentially be error prone. so im inclined to leave it how it is. (its like this in a few other places at least)